share/revng/test/tests/abi/build_abi_test_binaries_with_gcc.sh

```diff
@@ -29,6 +29,10 @@ mkdir -p "${OUTPUT_DIRECTORY}"
   ${CFLAGS} -O3 -ggdb3 \
   "${INPUT_DIRECTORY}/functions.c" \
   -o "${OUTPUT_DIRECTORY}/functions"
+"${TRIPLE}gcc" \
+  ${CFLAGS} -O3 \
+  "${INPUT_DIRECTORY}/functions.c" \
+  -o "${OUTPUT_DIRECTORY}/functions-no-debug"
```

Comment:
> It doesn't test anything but the fact that we don't crash during the conversion

I'm afraid we're making the time of the test suite explode.
How many binaries are we sending down the decompilation pipeline? Is there a real benefit in testing this?

We're supposed to test the ABI here, not have a bunch of diverse binaries for the sake of it.

Reply:
Actually, I just forgot to remove this (I only removed revng-counterpart).

It doesn't work. MSVC ones assert in `DetectABI`.

The whole point I even added this was because I was experiencing similar problems when doing post-dla-conversion analysis. So having this as a reliable way to reproduce problems helped with debugging a lot.

TL;DR: I didn't mean to push this in, my bad.